src/components/event/EventActiveStatus/Event-active-status.js

```diff
@@ -7,13 +7,14 @@ import eventStatusIcon from "../../../constants/eventStatusIcon";
 import SimpleModalWithDetails from "../../helpers/simple-modal-with-details";
 import RenderEventButton from "./RenderEventButton";
 
+const ARE_YOU_SURE = "Are you sure?";
```

Comment:
Extract this constant variable into separate file with other constants

Reply:
done